app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -438,10 +438,14 @@ function AcuantCapture(
     const { image, cardtype, dpi, moire, glare, sharpness } = nextCapture;
     const isAssessedAsGlare = glare < glareThreshold;
     const isAssessedAsBlurry = sharpness < sharpnessThreshold;
+    const isUnsupportedDocument = cardtype !== 1;
     const { width, height, data } = image;
 
     let assessment: AcuantImageAssessment;
-    if (isAssessedAsGlare) {
+    if (isUnsupportedDocument) {
+      setOwnErrorMessage(t('doc_auth.errors.card_type'));
+      assessment = 'unsupported';
```

Comment:
Can we add test coverage for this scenario resulting in the message being shown to the user and the relevant event metadata being logged?

Reply:
thinking about the same.

Reply:
Good point!  I added tests.